spec/Statements.tex

```diff
@@ -238,6 +239,48 @@ \section{The Swap Statement}
 \end{chapel}
 \end{example}
 
+\section{The I/O Statement}
+\label{The_IO_Statement}
+\index{IO!statement}
+\index{statements!IO}
+\index{IO!operator}
+\index{operators!IO}
+
+The I/O operator indicates writing to the left-hand-side the value
```

Comment:
Isn't the whole point of the I/O operator that it can be either a read or a write?  (i.e., the reason we didn't use << or >> as in C++?)